### PR TITLE
DIV-3264: Update readme

### PR DESCRIPTION
**Pull request by Cotterale**
# Description

Update readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,12 +77,18 @@ To run all integration tests locally:
   * Using IntelliJ: edit Run Configuration and set Environment variables to `http_proxy=http://proxyout.reform.hmcts.net:8080;SPRING_PROFILES_ACTIVE=aat`
```
Comment (yarpo) [inline README.md]: can you please update `src/main/resources/example-application-aat.yaml` to src/main/resources/example-application-aat.yml`

(.yaml -> .yml) as we don't have yaml files in here :)

Comment (hosainnet) [reply README.md]: Fixed 👍

File: README.md [hunk 2]
```diff
@@ -72,24 +65,30 @@ To run all mutation tests execute the following command:
 
 ```
 
-### Integration tests
+**Integration tests**
 
 To run all integration tests locally:
 
-* Make a copy of `src/main/resources/example-application-aat.yaml` as `src/main/resources/application-aat.yaml`
+* Make a copy of `src/main/resources/example-application-aat.yml` as `src/main/resources/application-aat.yml`
```
Comment (yarpo) [inline README.md]: <3

Comment (hosainnet) [reply README.md]: @yarpo ship it!

**Pull request merged**